source/adios2/core/Engine.h

```diff
@@ -63,6 +63,8 @@ class Engine
      */
     IO &GetIO() noexcept;
 
+    Mode OpenMode() const noexcept;
```

Comment:
We usually do properties as `GetFoo()` rather than `Foo()` don't we?

Reply:
Good point.....Engine has the non-const GetIO, GetSync and GetDeferred .... while the more informative CurrentStep and OpenMode are const.